src/main/java/org/helpberkeley/memberdata/WorkflowParser.java

```diff
@@ -36,6 +36,8 @@ public abstract class WorkflowParser {
             "Driver {0}, delivery for {1} at line {2} occurs before pickup from {3} at line {4}.\n";
 
     protected final ControlBlock controlBlock;
+    // Keep the line numbers in sync with the workflow sheet.
+    // CsvToBeanBuilder swallows the header row and nextRow() pre-increments line number
     protected int lineNumber = 1;
```

Comment:
Was this already accounting for the missing header?

Reply:
Yes.  This workflow parser was already doing the right thing.  It was the restaurant template parser that had the off by one error.  I added this comments in both to try to clarify what is going on with the line numbering.